src/reclaim_python_sdk/utils/proof_utils.py

```
@@ -54,23 +54,26 @@ async def create_link_with_template_data(template_data: TemplateData) -> str:
-            logger.info('No beacon available for getting witnesses')
-            raise Exception('No beacon available')
-        
-        state = await beacon.get_state(epoch)
-        witness_list = fetch_witness_list_for_claim(state, identifier, timestamp_s)
-        witnesses = [w.id.lower() for w in witness_list]
-        return witnesses
+        async with httpx.AsyncClient() as client:
+            response = await client.get(DEFAULT_ATTESTORS_URL)
+            if response.status_code != 200:
+                response.read()
+                raise Exception(f"Failed to fetch witness addresses: {response.status_code}")
```

Ruff (0.15.5)

[warning] Abstract `raise` to an inner function

(TRY301)